src/api/test/models/event_test.rb

```diff
@@ -91,13 +91,10 @@ def groups_for_event(e)
 
   test 'cleanup job' do
     firstcount = Event::Base.count
+    puts firstcount
```

Comment:
I would add here what the number represent. Or drop it, if it's debug code;-)

Reply:
It is debug code, sorry! 🙈